src/main/solution_space/path_finder_test_system.py

```diff
@@ -127,22 +128,20 @@ def __get_all_possible_path_finders(self) -> List[IPathFinder]:
             path_finders.append(pf_subclass(self._graph, mv_subclass))
         return path_finders
 
-    def __create_user_vertex(self, test_input: TestInput) -> Vertex:
-        vertex = Vertex(self._graph, Code.from_source(test_input[TEST_INPUT.SOURCE_CODE], None, self._graph.task))
-        # Todo: init profile if it's None
+    def __create_user_trees(self, test_input: TestInput) -> Tuple[AnonTree, ast.AST]:
         code_info = CodeInfo(User(Profile(test_input[TEST_INPUT.AGE], test_input[TEST_INPUT.EXPERIENCE])))
-        vertex.add_code_info(code_info)
-        return vertex
-
+        code = Code.from_source(test_input[TEST_INPUT.SOURCE_CODE], rate=None, task=self._graph.task)
+        anon_tree = AnonTree(code.anon_tree, code_info)
```

Comment:
It will require a file path, but it will be in my PR too. Can we merge my PR and you will fix yours?

Reply:
Sure